.travis.yml

```diff
@@ -15,6 +15,6 @@ before_install:
 
 script:
 - make lint
-- make ci-build
 - make ci-test
-- ./formicactl/formicactl
+- make ci-build
```

Comment:
I needed to change that order recently because the CI failed. When the dependencies are properly set within the Makefile, all good. No strong feelings.